client/src/utils/pagination.js

```diff
@@ -0,0 +1,16 @@
+
+export const handlePageChange = ({ currentTarget: input, state }) => {
```

Comment:
I am not sure but I think this function is never called, I don't find the way to use it

Reply:
The goal of the function is only to update the state.pageNumber, this is is already done by the  handlePageChangeSubmission function

`this.setState({ pageNumber: pageNumber }, () => {
// Callback 
}`